py/torch_tensorrt/dynamo/conversion/ops_evaluators.py

```diff
@@ -42,12 +49,48 @@ def generic_evaluator(
     return target(*args)
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.arange.start_step)
+@dynamo_tensorrt_converter(
+    torch.ops.aten.arange.start_step, supports_dynamic_shapes=True
+)
 def aten_ops_arange_start_step(
     ctx: ConversionContext,
     target: Target,
     args: Tuple[Argument, ...],
     kwargs: Dict[str, Argument],
     name: str,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    # Case where inputs to arange are dynamic
+    if np.any([isinstance(tensor, TRTTensor) for tensor in args]):
```

Comment:
Consider switching to Python `any`, as in `any(isinstance(tensor, TRTTensor) for tensor in args)`, to avoid checking boolean value on Numpy array